app/ide-desktop/lib/content/src/index.ts

```diff
@@ -116,9 +118,23 @@ class Main {
                     options.groups.startup.options.entry.value !=
                         options.groups.startup.options.entry.default
                 ) {
-                    // TODO: authentication here
-                    // appInstance.config.email.value = user.email
-                    appInstance.run()
+                    const hideAuth = () => {
+                       const auth = document.getElementById('dashboard')
+                       const root = document.getElementById('root')
+                       if (auth) auth.style.display = 'none'
+                       if (root) root.style.display = 'block'
+                    }
+                    const props: AppProps = {
+                        logger,
+                        // This package is an Electron desktop app (i.e., not in the Cloud), so
+                        // we're running on the desktop.
+                        runningOnDesktop: true,
```

Comment:
Wait, `content` is meant to be run both in cloud and in electron, so I don't understand this comment.

Reply:
This PR is solely only for IDE. Entrypoint for dasahboard will be created in different PR

Reply:
Yeah the code runs in both, but we act slightly differently depending on the platform. For example, in Electron we use deep links (`enso://`) which we don't want in the browser (we want `http://`).